packages/procps/whattime.c.patch

```diff
@@ -0,0 +1,19 @@
+--- procps-3.3.17.orig/proc/whattime.c	2021-02-09 17:11:25.000000000 +0700
```

Comment:
Why copying https://github.com/vhqtvn/termux-packages/blob/for-merge-20210528/packages/procps/proc-whattime.c.patch ?